typings/index.d.ts

```diff
@@ -3810,7 +3810,7 @@ export type MessageComponentType = keyof typeof MessageComponentTypes;
 export type MessageComponentTypeResolvable = MessageComponentType | MessageComponentTypes;
 
 export interface MessageEditOptions {
-  attachments?: MessageAttachment[];
+  attachments?: Array<MessageAttachment | never>;
```

Comment:
```suggestion
  attachments?: (MessageAttachment | never)[];
```
Djs doesn't use the `Array<Type>` form.